src/styles/product-card.scss

```diff
@@ -0,0 +1,107 @@
+.product-card {
+  width: 200px;
+  margin-right: 48px;
+  padding-bottom: 49px;
```

Comment:
This should be margin. In general, you should use padding when you want to add space within an element and use margin when you want to add space outside an element.